staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/conversion_test.go

```diff
@@ -76,8 +76,9 @@ func TestConvertSliceStringToDeletionPropagation(t *testing.T) {
 
 	for _, tc := range tcs {
 		var dp v1.DeletionPropagation
-		if err := v1.Convert_Slice_string_To_v1_DeletionPropagation(&tc.Input, &dp, nil); err != nil {
-			t.Errorf("Convert_Slice_string_To_v1_DeletionPropagation(%#v): %v", tc.Input, err)
+		dpPtr := &dp
```

Comment:
Why do you need it? Why you can't have &(&dp) below?

Reply:
&(&dp) will report an error: cannot take the address of &dp